mitiq/interface/mitiq_qiskit/tests/test_conversions_qiskit.py

```diff
@@ -153,7 +154,10 @@ def test_bell_state_to_from_circuits():
     """
     qreg = cirq.LineQubit.range(2)
     cirq_circuit = cirq.Circuit(
-        [cirq.ops.H.on(qreg[0]), cirq.ops.CNOT.on(qreg[0], qreg[1])]
+        [
```

Comment:
In my test, I do not see this correction. This applies to all explosions of arrays into multiple lines.